src/Sulu/Bundle/AdminBundle/Resources/js/containers/Form/fields/ResourceLocator.js

```diff
@@ -89,6 +68,43 @@ class ResourceLocator extends React.Component<FieldTypeProps<?string>> {
         onFinish();
     };
 
+    regenerateUrl = () => {
```

Comment:
Would call it just `generateUrl`, because when being called for the first time, it is not a regeneration.